code/modules/vore/fluffstuff/guns/sickshot.dm

```diff
@@ -1,7 +1,7 @@
 // -------------- Sickshot -------------
 /obj/item/weapon/gun/energy/sickshot
-	name = "\improper MPA6 \'Sickshot\'"
-	desc = "Need to stun someone? Don't mind having to clean up the mess afterwards? The Sickshot is the answer to your prayers. \
+	name = "energy revolver"
```

Comment:
emetic energy revolver

Reply:
Mate, I confused it with an energy gun but in revolver style when I seen it.